src/drivers/uavcan/uavcan_main.hpp

```diff
@@ -265,7 +265,7 @@ class UavcanNode : public px4::ScheduledWorkItem, public ModuleParams
 
 	uORB::SubscriptionInterval	_parameter_update_sub{ORB_ID(parameter_update), 1_s};
 	uORB::Subscription _vcmd_sub{ORB_ID(vehicle_command)};
-	uORB::Subscription _param_request_sub{ORB_ID(uavcan_parameter_request)};
+	uORB::SubscriptionInterval _param_request_sub{ORB_ID(uavcan_parameter_request), 100_us};
```

Comment:
This looks fine, but I'd assume that the correct value would be 1000_us . 10000_us, based on the picture and the comment " if FPS > 1000, there might be chance of corrupted frames". I assume that 100_us does the trick, but actually what it needs to do (and what the sleep() did before) is to really set it so that it limits the FPS to the maximum what you wanted :)